docs/forms/FORM_STRUCTURE.md

```diff
@@ -176,10 +156,11 @@ ngOnInit() {
               [ngTemplateOutletContext]="formsPocPresContext$ | async">
 </ng-container>
 ```
-* __Presenter__ class
-   * Here we have to create the formGroup/formArray/formControl object
-   * Provide [NG_VALUE_ACCESSOR](https://angular.io/api/forms/NG_VALUE_ACCESSOR) - used to provide a [ControlValueAccessor](https://angular.io/api/forms/DefaultValueAccessor) for form controls, to write a value and listening to changes on input elements.
-   * Provide [NG_VALIDATORS](https://angular.io/api/forms/NG_VALIDATORS) This is an [InjectionToken](https://angular.io/api/core/InjectionToken) for registering additional synchronous validators used with forms.
+
+__Presenter class:__
+   * Here we have to create the `formGroup`/`formArray`/`formControl` object.
+   * Provide [NG_VALUE_ACCESSOR](https://angular.io/api/forms/NG_VALUE_ACCESSOR) - used to provide a [ControlValueAccessor](https://angular.io/api/forms/DefaultValueAccessor) for form controls, to write a value and listen to changes on input elements.
+   * Provide [NG_VALIDATORS](https://angular.io/api/forms/NG_VALIDATORS) - this is an [InjectionToken](https://angular.io/api/core/InjectionToken) for registering additional synchronous validators used with forms.
 ```typescript
 // in presenter class
 @Component({
```

Comment:
Should we keep an example like that in the doc 
or have it in the showcase and add a link to it here